about.html

```diff
@@ -0,0 +1,123 @@
+					</ul>
+					<div class="close">
+						<i class="fa fa-times"></i>
+					</div>
+				</nav>
+			</div>
+			</div>
+		</header>
+		<main id="main">
+			<section class="headline">
+				<div class="our-terms">
+					<div class="our-term-main-container">
+						<p class="slogan">" Every Child Deserves a Future."</p>
+						<div class="main-tilte">
+              <h1>Adoptive families need community in 2021</h1>
+						</div>
+					</div>
+				</div>
+				<div class="detail-container">
+					<div class="bg-color"></div>
+					<div class="detail">
+						<p>
+              Yet, the number of adoptions has never kept up with number of children who desperately need a home.
+              At Adopt a Love Story, we recognize that each adoption tells the story of a mom, who despite all other circumstances or challenges, chooses to give her baby life. In doing so she writes a love story of her own, a story of potential and promise for her child.
+              When an adoptive family enters the picture they are entrusted with that love story. And this is where you join the story, by empowering adoptive families to bring their children home.
+						</p>
+					</div>
+					<div class="work-time">
+						<span>Please contact us per Email for more questions about Honestly adoption Company!</span>
+						<span>moiserushanika2006@gmail.com</span>
+					</div>
+				</div>
+			</section>
+			<section class="about-adoption">
+        <h2 class="about-title">Understanding Adoption, A Developmental Approach</h2>
+        <div class="about-adoption-detail"><p>We provide the resources and platform to empower families to raise funds for their adoption, and engage their community with the powerful story of why they are choosing adoption. As one of the purest expressions of the love and compassion of the Gospel, adoption has always provided the life-altering solution to this need.</p></div>
+        <div class="adoption-logo">
+          <img class="logo-img" src="images/Honestly+Adoption+Logo.png" alt="">
+        </div>
+      </section> 
+			<section class="our-family">
+        <h2 class="about-title">Discover a Story to Join</h2>
+        <div class="about-familly-detail about-adoption-detail">If you’re already on your adoption journey, or just getting ready to start, you can apply to be an Adopt a Love Story family by filling out a quick application.</div>
+        <ul class="family-images-container">
+
+          <li class="family-img">
+            <a href="#">
+              <img class="fimg" src="images/family_1.jpeg" alt="">
+              <div class="family-content">
+                <div class="bg"></div>
+                <div class="content-title">
+                  <h3 class="name">The Williams Family Love Story</h3>
+                </div>
+              </div>
+            </a>
+          </li>
+          <li class="family-img">
+            <a href="#">
+              <img class="fimg" src="images/family_2.jpeg" alt="">
```

Comment:
- [ ] Well done you added `alt` attribute but you need to add a text that can be displayed when image is not visible do it everywhere you have that attribute